src/style.css

```diff
@@ -0,0 +1,122 @@
+
+body,
+p {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 0 16px;
+  width: 166px;
+  height: 406px;
+  background: #fff ;
+  border: 1px solid  #f3f3f3;
+  border-radius: 5px;
+
+}
+
+.card__mac {
+  align-self: center;
+
+  width: 160px;
+  height: 134px;
+  padding: 32px 0 40px;
+
+}
+
+.card__product {
+  padding-bottom: 4px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__code {
+  padding-bottom: 16px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  padding-bottom: 16px;
+}
+
+.card__price-price {
+  align-items: flex-end;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__price-numbers {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__button {
+  box-sizing: border-box;
+  display: flex;
+  width: 166px;
+  height: 40px;
+  background: #00acdc;
+  border-radius: 5px;
+  text-transform: uppercase;
```

Comment:
u have already wrote in uppercase manualy, I would leave only option in styles

Reply:
ok .I will remember for the future.